en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md

```diff
@@ -0,0 +1,134 @@
+
+    !!! note 
+        
+        Assigning shard IDs to nodes allows the system to identify each unique node when assigning parts of the 
+        aggregation. If the shard IDs are not assigned, system uses the unique node IDs (defined in step 1) for this purpose.
+        
+    - For node 1:
+        ``` yaml
+        siddhi:
+          properties:
+            partitionById: true
+            shardId: wso2-sp-analytics-1
+        ```
+    - For node 2:
+    ``` yaml
+    siddhi:
+      properties:
+        partitionById: true
+        shardId: wso2-sp-analytics-2
+    ```
+    
+    !!! note 
+    
+        - To maintain data consistency, do not change the shard IDs after the first configuration
+        - When you enable the aggregation partitioning feature, a new column ID named SHARD_ID is introduced to the 
+        aggregation tables. Therefore, you need to do one of the following options after enabling this feature to 
+        avoid errors occuring due to the differences in the table schema.
+            - Delete all the aggregation tables for SECONDS, MINUTES, HOURS, DAYS, MONTHS, YEARS. 
+            - Edit the aggregation tables by adding a new column named SHARD_ID, and specify it as a primary key.
+
+3. Configure a database, and then update the default configuration for the APIM_ANALYTICS_DB data source with 
+parameter values suitable for your requirements in `<APIM_ANALYTICS_HOME>/conf/worker/deployment.yaml`
+
+   ``` yaml
+    
+   - name: APIM_ANALYTICS_DB
+         description: "The datasource used for APIM statistics aggregated data."
+         jndiConfig:
+           name: jdbc/APIM_ANALYTICS_DB
+         definition:
+           type: RDBMS
+           configuration:
+             jdbcUrl: "jdbc:mysql://localhost:3306/APIM_ANALYTICS_DB_1?useSSL=false"
+             password: pass
+             username: root
+             driverClassName: com.mysql.jdbc.Driver
+             maxPoolSize: 50
+             idleTimeout: 60000
+             connectionTestQuery: SELECT 1
+             validationTimeout: 30000
+             isAutoCommit: false
+         
+   ```  
+   
+   - If you are configure analytics for WSO2 Micro Gateway, import the 
+   appropriate DB script from `<APIM_ANALYTICS_HOME>/wso2/worker/dbscripts/apimgt/`
+
+!!! note 
+
+    As explained in above the events are processed in multiple active nodes. Eventhough this is usually a stateful 
+    operation, you can overcome the node-dependent calculations via distributed aggregation. This allows WSO2 API-M to 
+    execute APIM Analytics scripts that depend on incremental distributed aggregation.
+    
+    However, an active-active deployment can affect alerts because alerts also depend on some in-memory stateful 
+    operations such as windows. Due to this, alerts can be generated based on the events received by specific node. Thus 
+    the alerts are node-dependent, and you need to disable them to run scripts with distributed incremental aggregation.
```

Comment:
shall we just mention that some alert types only will be affected?  I don't think we need to mention anything about distributed incremental aggregation.

Reply:
fixed